inference-engine/src/mkldnn_plugin/nodes/psroi.cpp

```diff
@@ -7,7 +8,13 @@
 #include <vector>
 #include <string>
 #include <algorithm>
+#include <mkldnn_types.h>
 #include "ie_parallel.hpp"
+#include "c_types_map.hpp"
+#include "bfloat16_utils.hpp"
+#include "utils/bfloat16.hpp"
```

Comment:
If the bfloat16 class is used, the mkldnn converters are not needed, so there is no reason to include mkldnn converters headers.

Reply:
done